### ISSUE TITLE
SKPhoto.photoWithImageURL加载来自网络的图片时，HUD图片只会显示在第一张图上

### ISSUE THREAD

**Issue by googlb** [suzuki-0000/SKPhotoBrowser#96]
SKPhoto.photoWithImageURL加载来自网络的图片时，HUD加载中图片只会显示在第一张图上

**Comment by krummler** [comment]
Hello,

could you show an example how to reproduce this problem?

**Comment by googlb** [comment]
![1](https://cloud.githubusercontent.com/assets/2196880/15315127/61889052-1c48-11e6-9880-c763457623e0.png)
![2](https://cloud.githubusercontent.com/assets/2196880/15315132/6733907e-1c48-11e6-91de-a10cb46b9bc8.png)

![3](https://cloud.githubusercontent.com/assets/2196880/15315135/6dd51f7e-1c48-11e6-9a5e-a0fa9690fe83.png)
I use this func loading image from url
`SKPhoto.photoWithImageURL(url)`
I found that when loading multiple images from the network, The HUD will only appear on the first page, I don't know if you are  designed like that, I think if each page has its own HUD, when the image is  loading it show and hide after loading, the user experience may be better

**Comment by orazz** [comment]
I have same problem

**Comment by orazz** [comment]
```
if !flag {
            **if photo.underlyingImage == nil {**
                indicatorView.startAnimating()
            **}**
            photo.loadUnderlyingImageAndNotify()
        } else {
            indicatorView.stopAnimating()
        }
```

**Comment by suzuki-0000** [comment]
this problem is fixed at v4.0.0. please re-open when trouble happen
